docs/overview.md

```diff
@@ -1092,7 +1048,7 @@ if err == nil {
 }
 // (B) - typo in "==", change to "!="
 if err != nil {
-	return nil
+	return err
```

Comment:
Consider to update it here also https://github.com/go-critic/go-critic/blob/master/lint/nilValReturn_checker.go#L31

Reply:
@cristaloleg Done. I have updated the PR.